qgreenland/config/helpers/layers/sea_ice_age_params.py

```diff
@@ -0,0 +1,118 @@
+import datetime as dt
```

Comment:
What do you think about calling this module something like `get_sea_ice_age_params.py`? or `fetch_...`

Reply:
Renamed this `make_sea_ice_age_params.py`